docs/Installation/Installation_Guide.rst

```diff
@@ -21,74 +21,68 @@ public cloud resources, the following sizing requirements are assumed good for
 other cloud Kubernetes deployments, where such resources are preinstalled and
 available on demand.
 
-####################
 Setting Up a Cluster
 ####################
 
-****************************
 General Cluster Requirements
 ****************************
 
 Setting up a testing cluster requires at least three machines (these can be
-VMs) running CentOS_ 7.4 (The recommended mimimum for Zenko production service
-is five server nodes with three masters/etcds, but for testing and
-familiarization, three masters and three nodes is fine). You must have SSH
+VMs) running CentOS_ 7.4 (or higher)(The recommended mimimum for Zenko
+production service is five server nodes with three masters/etcds, but for
+testing and familiarization, three masters and three nodes is fine). You must
+have SSH
 access to these machines and they must have SSH access to each other. (You
 can copy SSH credentials from one machine to the next and log in once to
 ensure each machine has been added to the others' recognized hosts lists).
 Each machine acting as a Kubernetes_ node must also have at least one disk
 available to provision storage volumes.
 
+Once you have set up a cluster, you cannot change the size of the machines on
+it.
+
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-
 Sizing
 ======
 
-Sizing for Metal K8s
---------------------
-
--  Cores
+The following sizes have been for Zenko instances tested on live systems using
```

Comment:
"The following size for Zenko instances..."? Sounds kinds weird the way it is

Reply:
Good catch.